src/App.vue

```diff
@@ -33,6 +34,10 @@ const showRequestPage = computed(() => {
   return requestStore.areRequestsPendingForApproval
 })
 
+const showWalletButton = computed(() => {
+  return !app.expandWallet && app.validAppMode !== AppMode.Widget
```

Comment:
you can change the second condition to 

`app.validAppMode === AppMode.FullUI`

Reply:
there is no `NoUI`?

Reply:
Will be there for gaming SDK I guess, not sure

Reply:
ok, I guess let's keep it like this